tests/data/HC-19/report_edited_sha_bad/report.yaml

```diff
@@ -0,0 +1,100 @@
+apiversion: v1
+kind: verify-report
+metadata:
+    tool:
+        verifier-version: 1.9.0
+        reportDigest:  uint64:17691208852129424495
+        profile:
+            VendorType: community
```

Comment:
This chart is from `partners`, how come the VendorType is `community` here?

Reply:
I think that was the edit I did to cause the sha to be wrong but I put it back and changed something else.